e2e/env/env.go

```diff
@@ -61,11 +61,9 @@ func singularityEnv(t *testing.T) {
 		t.Run(currentTest.name, test.WithoutPrivilege(func(t *testing.T) {
 			args := []string{"exec", currentTest.image, "env"}
 
-			cmd := exec.Command(testenv.CmdPath, args...)
-			cmd.Env = append(os.Environ(), currentTest.env...)
-			b, err := cmd.CombinedOutput()
+			cmd := exec.Command(testenv.CmdPath, args..., append(os.Environ(), currentTest.env...))
 
-			out := string(b)
+			out := cmd.Run(t).Stdout()
 			t.Logf("args: '%v'", strings.Join(args, " "))
 			t.Logf("env: '%v'", strings.Join(cmd.Env, " "))
 			t.Log(out)
```

Comment:
it would be a good idea to specify that this the standard output. Something like:

```
Command: ...
Environment: ...
Standard output: ...
Standard error: ...
```